.github/workflows/visual_tests.yml

```diff
@@ -64,22 +64,50 @@ jobs:
     if: ${{ needs.visual_tests.result == 'success' }}
     runs-on: ubuntu-latest
     steps:
-      - name: Post results comment
-        uses: thollander/actions-comment-pull-request@v2
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          message: |
-            ### Percy Visual Test Results
+      - name: Define comment body
+        run: |
+            echo "### Percy Visual Test Results
+        
             **Percy Dashboard:** [View Detailed Report](${{ needs.visual_tests.outputs.percy_url }})
             
             **Environment:**
             - **Node.js Version:** 18.x
             - **OS:** Ubuntu-latest
-
+            
             **Instructions for Reviewers:**
             - Click on the [Percy Dashboard](${{ needs.visual_tests.outputs.percy_url }}) link to view detailed visual diffs.
             - Review the visual changes highlighted in the report.
-            - Approve or request changes based on the visual differences.
-          comment_tag: execution
-          mode: recreate
+            - Approve or request changes based on the visual differences." > comment_body.md
+        
+      - name: Create or update comment
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const fs = require('fs');
+            const commentBody = fs.readFileSync('comment_body.md', 'utf8');
+            
+            const { data: comments } = await github.rest.issues.listComments({
```

Comment:
It seems this listComments returns paginated data up to 30 results https://docs.github.com/en/rest/issues/comments?apiVersion=2022-11-28. In Kolibri we use [this function](https://github.com/learningequality/kolibri/blob/5582278a1d4179bc2191e9545275a4d93b647bf1/.github/githubUtils.js#L116) to find the generated comment, and then in the workflow we just import the file, and return the [function result](https://github.com/learningequality/kolibri/blob/5582278a1d4179bc2191e9545275a4d93b647bf1/.github/workflows/pr_build_comment.yml#L38). I think we can use a similar strategy moving this code to a file `.github/githubUtils.js` here in KDS to have the benefits of linting and code formatting.

Reply:
Sure. I'll change the current setup to something similar to what is used in Kolibri then 👍